### ISSUE TITLE
Fix Invalid Header filed

### ISSUE THREAD

**Issue by raphiz** [raphiz/nele#7]
```
Die Kontrolle der Nachricht ergab:
    Missing required header field: "Date"
```
...
```
First upstream SMTP client IP address: [2a00:d70:0:e::314]
  mxout014.mail.hostpoint.ch
According to a 'Received:' trace, the message apparently originated at:
  [46.127.178.62], mxout014.mail.hostpoint.ch mxout014.mail.hostpoint.ch
  [IPv6:2a00:d70:0:e::314] using TLSv1.2 with cipher
  ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits) No client certificate requested

Return-Path: <...>
From: "open\HSR Newsletter" <...>
Subject: ...
The message has been quarantined as: g/badh-...
```

**Comment by raphiz** [comment]
Will be fiexd with #1 - I have to release v0.2....

**Comment by raphiz** [comment]
Fixed in Nele v0.3.0
